services/src/northbound/src/checkstyle/checkstyle-suppress-known-issues.xml

```diff
@@ -41,6 +41,12 @@
     <suppress files="src/main/java/org/openkilda/northbound/controller/SwitchController.java" lines="24" checks="CustomImportOrder"/>
     <suppress files="src/main/java/org/openkilda/northbound/controller/SwitchController.java" lines="25" checks="CustomImportOrder"/>
     <suppress files="src/main/java/org/openkilda/northbound/controller/SwitchController.java" lines="39" checks="CustomImportOrder"/>
+    <suppress files="src/main/java/org/openkilda/northbound/controller/SwitchController.java" lines="40" checks="CustomImportOrder"/>
```

Comment:
Please, try to fix code issues in the code you modified, if you don't touch some lines - in that case you might update this file to ignore them

Reply:
Fixed import issues and removed this code.